src/cdk/tree/tree.spec.ts

```diff
@@ -0,0 +1,139 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {Component, ViewChild} from '@angular/core';
+
+import {CollectionViewer, DataSource} from '@angular/cdk/collections';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {Observable} from 'rxjs/Observable';
+import {combineLatest} from 'rxjs/observable/combineLatest';
+import {map} from 'rxjs/operator/map';
+
+import {TreeControl} from './control/tree-control';
+import {FlatTreeControl} from './control/flat-tree-control';
+import {CdkTreeModule} from './index';
+import {CdkTree} from './tree';
+
+
+describe('CdkTree', () => {
```

Comment:
In a follow-up PR, it would be good to explore creating a syntax-sugar way of testing tree content. The table, for example, has this:
```ts
      expectTableToMatchContent(tableElement, [
        ['Column A', 'Column B', 'Column C'],
        ['a_1', 'b_1', 'c_1'],
        ['a_2', 'b_2', 'c_2'],
        ['a_3', 'b_3', 'c_3'],
      ]);
```

For the tree, we'd need to capture level and expansion state